test/writer.typst

```diff
@@ -104,7 +104,6 @@
 }
 #show: doc => conf(
   title: [Pandoc Test Suite],
-  subtitle: [A set of tests for Pandoc],
```

Comment:
I'm confused about why this test output changes.

Reply:
I reverted a change I made.
I did not realised the native test did not include subtitle, so adding it would cause the test suite to fail.